extensions/google-shopping-ads/index.md

```diff
@@ -61,10 +61,14 @@ To add the Google API key:
 
     ![Add the API Key](images/config-api-key.png)
 
+    {:.bs-callout .bs-callout-info}
+You can also [Enable Service Logs](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) during this step. This option is disabled by default and should only be used when needed for troubleshooting, as continued logging will negatively impact performance.
```

Comment:
Links does not work. Not in production: https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html
Also would check if you need to indent the content for the note to properly align under the indented note code. If it should not indent, remove indent from note code.

Reply:
Also if this is strictly for version 2.0.0, may need to note that.

Reply:
Removed the first link comment as the merchdocs 2.0 PR is not merged yet. 👍